specs/honest-challenger-fdg.md

```diff
@@ -64,7 +64,13 @@ then it must dispute claims positioned at odd depths in the game tree.
 Otherwise, it disputes claims positioned at even depths in the game tree.
 This means an honest challenger only responds to claims made by the opposing team.
 
-The next step is to determine whether the claim has a valid commitment (i.e. `ClaimHash`).
+The next step is to determine if the claim, now known to be for the opposing team,
+disputes another claim the honest challenger _agrees_ with.
+An honest challenger agrees with a claim iff every other claim along its path to the
+root claim commits to a valid `ClaimHash`. Thus, an honest challenger avoids supporting
+invalid claims on the same team.
```

Comment:
We should make it clear that the honest challenger does not counter a claim at all if it disagrees with the path of claims leading to that claim. The current wording sounds like it would just disagree and therefore attack.

Reply:
Rewritten to make that clearer.